packages/web/pages/stake/index.tsx

```diff
@@ -8,20 +8,34 @@ import { AlertBanner } from "~/components/alert-banner";
 import { MainStakeCard } from "~/components/cards/main-stake-card";
 import { StakeDashboard } from "~/components/cards/stake-dashboard";
 import { StakeLearnMore } from "~/components/cards/stake-learn-more";
+import { EventName } from "~/config";
 import { useAmountConfig, useFakeFeeConfig } from "~/hooks";
+import { useAmplitudeAnalytics } from "~/hooks";
 import { useFeatureFlags } from "~/hooks/use-feature-flags";
 import { ValidatorNextStepModal } from "~/modals/validator-next-step";
 import { ValidatorSquadModal } from "~/modals/validator-squad";
 import { useStore } from "~/stores";
 
+type AmountDefault = "half" | "max" | "input";
+
+interface Properties {
+  amountDefault: AmountDefault;
+  amount: string;
+  amountUSD: string | undefined;
+}
```

Comment:
FYI we have a properties type defined in v2 for properties sent to amplitude. IMO we should isolate amplitude types there as much as possible, to avoid having amplitude types be confused with actual business-logic data types.